utils_test.go

```diff
@@ -118,6 +119,28 @@ func TestGetFullQueryGzipped(t *testing.T) {
 	}
 }
 
+func TestGetFullQueryLZ4(t *testing.T) {
```

Comment:
Add negative tests when the query contains `decompress=1`, but the query doesn't conatin valid `lz4` request or contains truncated `lz4` request.